src/components/Blog/Post/index.tsx

```diff
@@ -83,8 +83,10 @@ const Post: React.FC<IBlogPostData> = ({
               />
             </div>
           </div>
-
-          <HeroPic picture={picture} pictureComment={pictureComment} />
+          <HeroPic
+            picture={picture?.gatsbyImageData}
+            pictureComment={pictureComment}
+          />
```

Comment:
My first inclination seeing this is that it'd be better if `HeroPic` takes `picture` directly and processes it inside the component, but that's at best a nitpick and there may be a good reason to do otherwise.

Reply:
To make the typescript less complex, I set `IBlogPostHeroPic.picture` (aka `HeroPic`'s defined prop value for `picture`) to only take a `IGatsbyImageData` object instead of entirety of `picture` which now includes `gatsbyImageData` and the `fields.sourcePath`.